telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/Generated/GeneratedCode.cs

```diff
@@ -16,7 +15,7 @@
 /// This file is generated from https://github.com/aws/aws-toolkit-common/tree/master/telemetry
 /// --------------------------------------------------------------------------------
 
-namespace Amazon.AwsToolkit.Telemetry.Events.Tests.Generated
+namespace Amazon.AwsToolkit.Telemetry.Events.Generated
```

Comment:
You don't need to review the changes to this file. It is produced by the generator and used in some tests.
This line here shows how the bug wasn't caught in the tests, because the supplemental generated and this generated file were produced to the same namespace, meaning the supplemental code had direct access to types (like Result) from this file.